plugins/com.google.cloud.tools.eclipse.ui.util/META-INF/MANIFEST.MF

```diff
@@ -9,7 +9,8 @@ Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Bundle-ActivationPolicy: lazy
 Require-Bundle: org.eclipse.ui.workbench,
- org.eclipse.ui
+ org.eclipse.ui,
+ org.eclipse.search
```

Comment:
Is the only reason we're pulling in this search package to get the image? If so, I wonder if we might be better off copying it, and then we can remove the logic to choose between GIF and PNG.